app/src/androidTest/java/com/sdp/movemeet/Activity/UploadActivityActivityTest.java

```diff
@@ -72,6 +72,11 @@ public static ViewAction forceDoubleClick() {
         };
     }
 
+    @Test
+    public void do_nothing() {
```

Comment:
Why is there this empty test?

Reply:
I added this test when I commented the rest of the tests so that this class still runs, but I should delete it now indeed !